com/vorauszahlung_test.go

```diff
@@ -0,0 +1,48 @@
+package com_test
+
+import (
+	"github.com/go-playground/validator/v10"
+	"github.com/hochfrequenz/go-bo4e/com"
+	"github.com/hochfrequenz/go-bo4e/enum/waehrungscode"
+	"github.com/shopspring/decimal"
+	"time"
+)
+
+func (s *Suite) Test_Failed_VorauszahlungValidation() {
+	validate := validator.New()
+	invalidVorauszahlungMap := map[string][]interface{}{
+		"required": {
+			com.Vorauszahlung{
+				// no Betrag given
+			},
+		},
+	}
+	VerfiyFailedValidations(s, validate, invalidVorauszahlungMap)
+}
+
+func (s *Suite) Test_Successful_VorauszahlungValidation() {
+	validate := validator.New()
+	arbitraryReferenz := "foo"
+	arbitraryDate := time.Date(2022, 9, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
ist das arbitrary best practice?

Reply:
hätte es auch "random" oder "some" nennen können. kenne keine best practices in der richtung.
ich brauchte nur die variable um den pointer zu machen.